drv/stm32xx-sys/src/main.rs

```diff
@@ -255,20 +284,132 @@ impl idl::InOrderSysImpl for ServerImpl<'_> {
     ) -> Result<[u32; 3], RequestError<core::convert::Infallible>> {
         Ok(drv_stm32xx_uid::read_uid())
     }
+
+    fn control_exti_source(
+        &mut self,
+        rm: &RecvMessage,
+        disable_mask: u32,
+        enable_mask: u32,
+    ) -> Result<(), RequestError<core::convert::Infallible>> {
+        cfg_if! {
+            if #[cfg(feature = "exti")] {
+                Ok(())
+            } else {
+                Ok(())
```

Comment:
should this perhaps reply-fault if it's called when `sys` is built without the `exti` feature?

Reply:
the _ideal_ thing would probably be to make idol codegen aware of feature flags, so that the client stub isn't generated unless it's enabled? but let's figure that out later.

Reply:
It actually does reply-fault on line 433.

Reply:
yup, i think i left the comment before it did that ---carry on!